system/jlib/jlog.hpp

```diff
@@ -1449,11 +1449,20 @@ interface jlib_decl ILogAccessFilter : public IInterface
     virtual LogAccessFilterType filterType() const = 0;
 };
 
+typedef enum
```

Comment:
style: enum LogAccessReturnColsMode { }; is preferred - this style is old C style.